EventsExpress/ClientApp/src/reducers/config.js

```diff
@@ -1,17 +1,17 @@
 ﻿import initialState from '../store/initialState';
-import { GET_CONFIGS_PENDING, GET_CONFIGS_SUCCESS } from '../actions/config/get-configs-action'
+import { GET_CONFIG_PENDING, GET_CONFIG_SUCCESS } from '../actions/config/get-configs-action'
 
-export const reducer = (state = initialState.configs, action) => {
+export const reducer = (state = initialState.config, action) => {
     switch (action.type) {
-        case GET_CONFIGS_PENDING: return {
+        case GET_CONFIG_PENDING: return {
             ...state,
             isConfigsPending: false
```

Comment:
Do we use this state value anywhere? I guess no, cause as far as I see we set it here to the wrong 'false' instead of 'true', and never change. So, probably would be better to remove it generally